### ISSUE TITLE
The server bar side is completely messed up

### ISSUE THREAD

**Issue by l3andoer** [maenDisease/Solana#18]
please fix it this is the only theme I like!!! 

![ss1](https://github.com/user-attachments/assets/cef316e0-324a-49fd-bd5a-f998b76bf161)

**Comment by Landjager** [comment]
do you mean the whole bar alignment or only the color fade?
and could you link the css file?

**Comment by l3andoer** [comment]
> do you mean the whole bar alignment or only the color fade? and could you link the css file?

both the weird color fade at the bottom and the messed up alignment 
----------------------------------------------------------------------------------------------------------------------------------
CSS FILE:
@import url('https://fonts.googleapis.com/css2?family=Yanone+Kaffeesatz:wght@100;300;400;500;700&display=swap');
@import url('https://maendisease.github.io/Solana/Solana.css');
@import url('https://maendisease.github.io/BetterDiscordStuff/css/bdAddonMini.css');
@import url('https://maendisease.github.io/BetterDiscordStuff/Themes/Solana/addon/mac-titlebar.css');
@import url('https://mwittrien.github.io/BetterDiscordAddons/Themes/EmojiReplace/base/Apple.css');
@import url('https://maendisease.github.io/BetterDiscordStuff/css/bdAddonMini.css');
@import url('https://maendisease.github.io/BetterDiscordStuff/Themes/Solana/addon/mac-titlebar.css');
@import url('https://mwittrien.github.io/BetterDiscordAddons/Themes/EmojiReplace/base/Apple.css');

:root {
  --accent-hue: 222;
  --accent-saturation: 100%;
  --accent-brightness: 50%;
  --background-accent-hue: 344;
  --background-accent-saturation: 100%;
  --background-accent-brightness: 50%;
  --background-image: initial; /* url(https://site.com/foobar.png) | URL MUST BE A DIRECT LINK (ending in .jpg, .jpeg, .png, .gif) */
  --background-image-position: center;
  --background-image-size: cover;
  --background-image-blur: 0px;
  --background-overlay-opacity: 80%;
  --background-overlay-color-lightness: 0;
  --background-hue-shift-strength: 100%;
  --window-background-image: initial;
  --window-background-image-position: center;
  --window-background-image-size: cover;
  --window-background-image-attachment: fixed;
  --window-opacity: 100%;
  --window-blur: 0;
  --window-friends-blur: 0;
  --window-margin: 16;
  --window-margin-left: 1;
  --window-border-radius: 20px;
  --window-background-lightness: 0;
  --window-accent-tint-strength: 5%;
  --chat-bubble: 1;
  --text-link-hue: 303;
  --chat-text-brightness: 0;
  --home-icon: initial;
  --favorites-icon: initial;
  --dock-blur: 0; /* Blur intensity of the dock. */
  --dock-border-radius: 1;
  --sidebar-width: 240px;
  --members-width: 240px;
  --sidebar-text-brightness: 0;
  --background-floating-blur: 20px;
  --floating-background-lightness: 0;
  --background-floating-opacity: 1;
  --enable-floating-transparency: 1;
  --custom-font-display: "";
  --custom-font-regular: Bebas Neue;
  --custom-font-primary: Yanone Kaffeesatz;
  --custom-font-code: "";
  --enable-blur: 1;
}

/* Any custom CSS below here */

**Comment by Landjager** [comment]
what about the metadata for eg. "@name" and "@author" and so on?
nvm i just copied it from another theme.

**Comment by l3andoer** [comment]
> what about the metadata for eg. "@name" and "@author" and so on? nvm i just copied it from another theme.

nice, you have any ideas on what could be wrong though?

**Comment by Landjager** [comment]
currently your issue doesnt exist for me. im playing with custom css and plugins right now trying to figure out what could be the reason for that

**Comment by l3andoer** [comment]
> currently your issue doesnt exist for me. im playing with custom css and plugins right now trying to figure out what could be the reason for that

yours doesn't have the weird alignment or the fade? Also thanks for trying to help me!

**Comment by Landjager** [comment]
yeah so if you dont really care about the Discovery button at the button you can just add this at the bottom of your theme. 
`.footer_aa1bff { display: none; }`
Otherwise I gotta look a bit deeper into that theme but the devtools somehow bug out on this theme lol

**Comment by l3andoer** [comment]
> yeah so if you dont really care about the Discovery button at the button you can just add this at the bottom of your theme. `.footer_aa1bff { display: none; }` Otherwise I gotta look a bit deeper into that theme but the devtools somehow bug out on this theme lol

this complete fixed it for me thank you man!
